docs/source/dataframe-api.rst

```diff
@@ -414,6 +414,15 @@ Rolling
 
 .. autofunction:: map_overlap
 
+Resampling
+~~~~~~~~~~
+
+.. currentmodule:: dask.dataframe.tseries.resample
+
+.. autoclass:: Resampler
```

Comment:
This class doesn't have a docstring, neither does the `_agg` method which seems to be used to generalize things.  We might want to add something small there, and also maybe make the `._agg` method non-private.

Reply:
Just pushed a commit which adds docstrings for `Resampler` and `Resampler._agg`. It seems the public `Resampler.agg` method should work for specifying general aggregations

```python
In [1]: from dask.dataframe.io.demo import make_timeseries

In [2]: ddf = make_timeseries()

In [3]: import numpy as np

In [4]: ddf.resample('1d').agg(np.sum).compute()
Out[4]:
                   id          x           y
timestamp
2000-01-31  8638696.0  47.077070 -134.875168
2000-02-01  8636932.0  78.899947  -22.346659
2000-02-02  8636624.0  32.193012  -18.677558
2000-02-03  8638606.0  70.468358  -10.982665
2000-02-04  8639956.0  44.161050  -77.516730
...               ...        ...         ...
2000-12-27  8639431.0  12.466025   62.642564
2000-12-28  8638456.0 -34.224802   57.942250
2000-12-29  8643282.0  20.416100   20.420697
2000-12-30  8643198.0  33.595944  -10.400613
2000-12-31        NaN        NaN         NaN

[336 rows x 3 columns]
```

Reply:
Oh grand